client/reader/feed-stream/index.jsx

```diff
@@ -1,94 +1,109 @@
-import { localize } from 'i18n-calypso';
```

Comment:
I think you could have made all the PRs build on top of each other e.g.

`trunk` <- `update/feed-stream-to-functional-component` <- `update/reader-stream-sidebar-using-prop` <- `add/reader-site-feed-sidebar`

I asume you can do this with github, I mean if you made the `update/reader-stream-sidebar-using-prop` branch explicitly depend on the `update/feed-stream-to-functional-component` branch. I don't think I've ever done it myself but it should be possible and that should hide this part of the diff.

Reply:
I did branch off one of those, but left the other as an independent change. In hindsight, it would have been better to create the branch sequence as you suggested! It was kind of gnarly getting this one reconciled once everything else was merged.